src/main/java/uk/nhs/digital/nhsconnect/nhais/model/edifact/PersonAddress.java

```diff
@@ -27,7 +27,7 @@ public String getKey() {
     @Override
     public String getValue() {
         String address = Stream.of(addressLine1, addressLine2, addressLine3, addressLine4, addressLine5)
-            .filter(Objects::nonNull)
+            .map(s -> s == null ? "" : s)
```

Comment:
maybe StringUtils.isEmpty or isBlank instead of '==' ?

Reply:
as above